File: my_test_apps/MyIntentService01/app/src/main/java/com/example/yks93/myintentservice01/MainActivity.java
package com.example.yks93.myintentservice01;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {


    private final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        Log.d(TAG, "onCreate: ");
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

}

File: my_test_apps/MyFragment01/app/src/main/java/com/example/yks93/myfragment01/MainActivity.kt
package com.example.yks93.myfragment01

import android.app.FragmentManager
import android.app.FragmentTransaction
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        addFragment()
    }

    private fun addFragment() {
        var frManager = getSupportFragmentManager()
        var frTransaction = frManager.beginTransaction()
        var sampleFragment = SampleFragment02()
        frTransaction.add(R.id.fragmentContainer, sampleFragment)
        frTransaction.commit()

    }

}
